src/router/index.js

```diff
@@ -13,6 +13,16 @@ Vue.use(Router);
 
 export default new Router({
   mode: 'history',
+  scrollBehavior: (to, from, savedPosition) => {
+    if (to.hash) {
```

Comment:
I would personally put savedPosition first if it doesn't create issues.
Reason is : you're navigating from a page with a hash, you should prob be scrolling where you were, not at the hash you arrived

Reply:
We did it this way because when loading a page with a hash with savedPosition first, it wasn't jumping to the hash since savedPosition was taking precedence

Reply:
savedPosition should take precedence over the hash 😄